charts/camunda-platform/charts/identity/templates/deployment.yaml

```diff
@@ -26,6 +26,36 @@ spec:
             value: "demo"
           - name: KEYCLOAK_USERS_0_ROLES_0
             value: "Identity"
+
+
+          {{- if .Values.global.operate.auth.identity.enabled }}
+          - name: KEYCLOAK_USERS_0_ROLES_1
+            value: "Operate"
+          - name: KEYCLOAK_INIT_OPERATE_SECRET
+            {{- if .Values.global.operate.auth.identity.existingSecret }}
+            valueFrom:
+              secretKeyRef:
+                {{- /*
+                    Helper: https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_secrets.tpl
+                    Usage in keycloak secrets https://github.com/bitnami/charts/blob/master/bitnami/keycloak/templates/secrets.yaml
+                    and in statefulset https://github.com/bitnami/charts/blob/master/bitnami/keycloak/templates/statefulset.yaml
+                */}}
+                name: {{ include "common.secrets.name" (dict "existingSecret" .Values.global.operate.auth.identity.existingSecret "context" $) }}
+                key: operate-secret
+            {{- else }}
+            valueFrom:
+              secretKeyRef:
+                {{- /*
+                    Helper: https://github.com/bitnami/charts/blob/master/bitnami/common/templates/_secrets.tpl
+                    Usage in keycloak secrets https://github.com/bitnami/charts/blob/master/bitnami/keycloak/templates/secrets.yaml
+                    and in statefulset https://github.com/bitnami/charts/blob/master/bitnami/keycloak/templates/statefulset.yaml
+                */}}
+                name: {{ include "identity.secretNameOperateIdentity" . }}
+                key: operate-secret
+            {{- end }}
+          - name: KEYCLOAK_INIT_OPERATE_ROOT_URL
+            value: {{ .Values.global.operate.auth.identity.operateRootUrl | quote }}
+          {{- end }}
           - name: SERVER_PORT
             value: "8080"
```

Comment:
Yeah TBH I thought it is not necessary as normal User to access identity :shrug: It worked for me with just port-forward to keycloak and operate. Is it not expected to work like that? :D

Reply:
BTW shouldn't be a problem as far as I can see, since the user need to port-forward any way the traffic and he have to choose a free local port right?